### ISSUE TITLE
WebDriver this.browser.getHTML is not a function

### ISSUE THREAD

**Issue by yusufilkeroguz** [Codeception/CodeceptJS#1431]
**NodeJS:** 8.12.0
**CodeceptJS:** ^2.0.2
**WebdriverIO:** ^5.2.5

When i use

```javascript
  I.grabHTMLFrom(locate('strong').inside('.headerSummary').first())
```

i got this error:

[1.basic:firefox]     [4] Error | TypeError: this.browser.getHTML is not a function
[1.basic:firefox]     Emitted | step.failed (I grab h t m l from ".//strong[ancestor::*[contains(concat(' ', normalize-space(./@class), ' '), ' headerSummary ')]][position()=1]")
[1.basic:firefox]     Emitted | step.finish (I grab h t m l from ".//strong[ancestor::*[contains(concat(' ', normalize-space(./@class), ' '), ' headerSummary ')]][position()=1]")
[1.basic:firefox]     [4] Error | TypeError: this.browser.getHTML is not a function

**Comment by xMutaGenx** [comment]
This is due to `commands are scoped to browser and element - no selector property on commands anymore` ref: https://github.com/webdriverio/webdriverio/blob/master/CHANGELOG.md
Webdriver Helper currently still uses browser scope instead of element scope for `getHTML()` command https://github.com/Codeception/CodeceptJS/blob/master/lib/helper/WebDriver.js#L766 You'll have to wait until this is updated, or contribute by forking and fixing this! Hope that helps.

**Comment by xMutaGenx** [comment]
Additionally.. it would look something like:
```
async grabHTMLFrom(locator) {
    const element = await this.browser.$(locator)
    const html = element.getHTML();
    this.debugSection('Grab', html);
    return html;
  }
```

However, the command should probably consider having the ability to use `includeSelectorTag` as per  https://webdriver.io/docs/api/element/getHTML.html

**Comment by DavertMik** [comment]
Should be fixed in 2.0.3

**Comment by sriguda-nbcuni** [comment]
TypeError: browser.getElementSize is not a function
TypeError: browser.elementHover is not a function

I am getting TypeError when I use this function in firefox, safari, IE.
It works fine on Chrome. Any idea about why it gives error on these browsers please.
